pages/windows/select-string.md

```diff
@@ -0,0 +1,26 @@
+# Select-String
+
+> Finds text in strings and files in PowerShell.
+> This command can only be used through PowerShell.
```

Comment:
We generally only allow 2 lines of page description + 1 for the link with some exceptions. I guess we can leave this as it is or else we can add this with the above line if necessary.

Reply:
It makes sense keeping as a separate line, but I can change if any other maintainer wants it.

Reply:
> We generally only allow 2 lines of page description + 1 for the link with some exceptions. I guess we can leave this as it is or else we can add this with the above line if necessary.

Not really, there are some edge cases when this is necessary, especially PowerShell-specific commands and Android shell commands which are only available under `adb shell`. You can see some examples here:

+ https://github.com/tldr-pages/tldr/blob/main/pages/windows/invoke-webrequest.md
+ https://github.com/tldr-pages/tldr/blob/main/pages/android/bugreportz.md

Reply:
> > We generally only allow 2 lines of page description + 1 for the link with some exceptions. I guess we can leave this as it is or else we can add this with the above line if necessary.
> 
> Not really, there are some edge cases when this is necessary, especially PowerShell-specific commands and Android shell commands which are only available under `adb shell`. You can see some examples here:
> 
> * https://github.com/tldr-pages/tldr/blob/main/pages/windows/invoke-webrequest.md
> * https://github.com/tldr-pages/tldr/blob/main/pages/android/bugreportz.md

Yeah, my comment is kinda outdated as some git pages, etc have 4 lines of descriptions as they require extra context.